### ISSUE TITLE
Missing anchor points on Mandaic contextual forms

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1504]
### Font
NotoSansMandaic-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/af306de71ce51b9ce75389b163026427953606cf/phaseIII_only/unhinted/ttf/NotoSansMandaic/NotoSansMandaic-Regular.ttf
Date: 2019-04-25

### Font version
Version 2.000

### Issue
Contextually wider or narrower forms of Mandaic letters are missing anchor points.

### Character data
ࡂࡈ࡙
U+0842 MANDAIC LETTER AG
U+0848 MANDAIC LETTER ATT
U+0859 MANDAIC AFFRICATION MARK

### Screenshot
![ࡂࡈ࡙](https://user-images.githubusercontent.com/1124347/57180456-a8381000-6e56-11e9-9d62-a414610605b5.png)
With 'calt' disabled:
![ࡂࡈ࡙](https://user-images.githubusercontent.com/1124347/57180457-a8381000-6e56-11e9-9fc8-172959a22fed.png)

**Comment by LornaSIL** [comment]
Is there any chance this will get fixed? This seems to be the only Mandaic font available and without these two characters working, it's really unusable.

**Comment by dscorbett** [comment]
To be clear, this bug report is about anchor points. The two letters ag and aṭ are just an example. You have found a different problem.

**Comment by punchcutter** [comment]
There are actually anchors on these glyphs, but the fontmake pipeline expects the script properties to also be set on the glyphs. That's a very easy fix. 

What @LornaSIL shows is something else, but I'm not sure what yet.

**Comment by LornaSIL** [comment]
Do I need to create a separate issue?

**Comment by punchcutter** [comment]
Yes, that's a different issue which so far I can't reproduce on my Win10.

**Comment by LornaSIL** [comment]
Ok, I installed the font linked to above and it doesn't have the problem I was seeing. The font I was using is also labeled as ver 2.000, but the version string is quite different. Here are the properties:
![image](https://user-images.githubusercontent.com/7071907/77964547-d7dcdb80-72a4-11ea-88e8-3923e57b4ed8.png)

So, the version linked to above seems to have resolved the issue I was seeing. I'm not actually sure where I downloaded the font from.

I'm sorry for the diversion.
